sympy/physics/units/quantities.py

```diff
@@ -21,9 +21,7 @@ class Quantity(AtomicExpr):
     """
 
     is_commutative = True
-    is_real = True
```

Comment:
these are redundant (as added tests show)

Reply:
The only assumption I see that relates to this is `_eval_is_positive`, which could return False if the scale factor isn't positive (though I don't know if complex scale factors are possible).